teamA/jeongHaeMan1/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "jeonghaeman1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^1.0.3",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql": "^2.18.1",
+    "mysql2": "^2.3.3",
```

mysql & mysql2 드라이버 패키지는 서로 동일한 기능을 하는 패키지 입니다. 따라서 둘 중 하나를 선택하여 설치를 진행해주세요.

```suggestion
    "mysql": "^2.18.1"
```